database/dbService.js

```diff
@@ -63,7 +85,7 @@ module.exports = {
             console.log('Database error:', err);
             if (err.code === 'PROTOCOL_CONNECTION_LOST') {
                 console.log("Reconnecting");
-                handleDisconnect();
+                tryConnect();
             } else {     
```

Comment:
My method was previously named handleDisconnect() but I forgot to rename the method calls.